KronorComponents/Common/EmbeddedPaymentViewModel.swift

```diff
@@ -153,7 +157,8 @@ class EmbeddedPaymentViewModel: ObservableObject {
                     )
                 case .payPal:
                     return await networking.createPayPalRequest(
-                        returnURL: self.returnURL,
+                        returnURL: self.intermediateRedirectURL,
+                        merchantReturnURL: self.returnURL,
```

Comment:
It looks a bit confusing that we pass `returnURL` to `merchantReturnURL` and not the other parameter.
Would it be viable to keep the returnURL how it behaved previously (as the merchant return url) and instead having an additional `intermediateRedirectURL` property for when additional redirect is needed?

Reply:
The end result is the same, we just need an intermediate step because PayPal has additional requirements after a payment is authorised, so we need to handle those in a page we can control. That page will redirect to the provided `returnURL`